src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,26 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { tabs } from '../../constants';
```

Comment:
You should import tabs inside App component and pass it like prop to tabs component due to task description
![image](https://github.com/mate-academy/react_tabs-js/assets/80700247/7338a87c-3eb4-495a-8059-ee35303eea0b)